client/src/components/Header.js

```diff
@@ -31,18 +32,19 @@ class Header extends Component {
     render() {
         var { auth: user } = this.props;
         return (
-            <nav>
-                <div className="nav-wrapper">
+            <div>
+                <div className="header">
                     <Link 
```

Comment:
I'd make this <div className="header"> the parent div, and get rid of the outermost div.